packages/app-migrations/lib/index.js

```diff
@@ -1,24 +1,26 @@
-var semver = require('semver');
-var async = require('async');
-var _ = require('lodash');
-var format = require('util').format;
-
-var debug = require('debug')('app-migrations');
+const semver = require('semver');
+const async = require('async');
+const _ = require('lodash');
+const format = require('util').format;
+const { log, mongoLogId } = require('@mongodb-js/compass-logging')('COMPASS-MIGRATIONS');
 
 function migrate(migrations) {
   return function(previousVersion, currentVersion, done) {
     var tasks;
     if (semver.lt(previousVersion, currentVersion)) {
       // pick migration tasks for upgrade
-      debug('upgrading from version', previousVersion, 'to version', currentVersion);
       tasks = _.pickBy(migrations, function(fn, version) {
         return semver.gt(version, previousVersion) &&
                semver.lte(version, currentVersion);
       });
       tasks = _.mapValues(tasks, function(fn) {
         return fn.bind(null, previousVersion, currentVersion);
       });
-      debug('executing migration steps for versions %j', _.keys(tasks));
+      log.info(mongoLogId(1001000070), 'Migrations', 'Considering upgrade migration', {
```

Comment:
What's with all these hardcoded numbers? Shouldn't we export some named constants from somewhere (probably @mongodb-js/compass-logging) to give these names?

Reply:
In fact all three the first params seem to be related so something like `log.info(...Logs.MIGRATIONS_CONSIDERING_UPGRADE_MIGRATION, { params go here })`. There are probably other alternatives as well.

Then if we encounter an id we know where to go look for it and it is probably easier to make sure that the ids are unique.

Reply:
What would be the benefit of that? We would essentially just be duplicating information about the log entry across multiple parts of the application, instead of having it be local to one single location

> Then if we encounter an id we know where to go look for it and it is probably easier to make sure that the ids are unique.

There’s a script that verifies that and provides the next free log id, just like the server’s script in their repo.